CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Added 
+
+- `attributes` field in `/list-resources` and `/list-subjects` endpoint
+
+### Changed
+
+- `attributes` field in `/check` endpoint
```

Comment:
Would be helpful to explain what specifically changed.

Reply:
Updated